lib/apps/customer_app/features/auth/ui/screens/reset.password.screen.dart

```diff
@@ -0,0 +1,156 @@
+import 'package:flutter/cupertino.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+import '../../../../../../shared/res/icons.dart';
+
+class ResetPassword extends StatelessWidget {
+  ResetPassword({super.key});
+  final GlobalKey<FormState> _formKey = GlobalKey<FormState>();
+  final TextEditingController _password = TextEditingController();
+  final TextEditingController _confirmPassword = TextEditingController();
+  final _obscureTextNotifier = ValueNotifier(true);
+  String passwordError = "password do not match";
+  void onPasswordSaved(String? password) {
+    _password.text = password!;
+  }
+
+  String? passwordValidator(String? password) {
+    if (password != null && password.length > 6 && _password.value.text != _confirmPassword.value.text) {
+      return null;
+    }
+    return 'Invalid password ' + passwordError;
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return SafeArea(
+      child: Scaffold(
+        body: Stack(children: [
+          Column(
+            mainAxisAlignment: MainAxisAlignment.center,
+            children: [
+              Padding(
+                padding: const EdgeInsets.only(bottom: 230.0),
+                child: SvgPicture.asset("assets/svg/key.svg"),
+              ),
+            ],
+          ),
+          Positioned(
+              right: 100,
+              top: 180,
+              child: Image(
+                image: AssetImage("assets/icon/key.png"),
+              )),
+          Column(
+            crossAxisAlignment: CrossAxisAlignment.start,
+            mainAxisAlignment: MainAxisAlignment.end,
+            children: [
+              Text(
+                "Reset \nPassword",
+                style: TextStyle(
+                  fontSize: 37,
+                  fontWeight: FontWeight.w700,
+                ),
+              ),
+              SizedBox(
+                height: 30,
+              ),
+              SizedBox(
+                height: 30,
+              ),
+              Form(
+                key: _formKey,
+                child: Column(
+                  children: [
+                    ValueListenableBuilder<bool>(
+                      valueListenable: _obscureTextNotifier,
+                      builder: (context, obscureText, child) {
+                        return TextFormField(
```

Comment:
the textfields on both forgot password and reset password screens have inputs with borders in all directions but it should only have at the bottom.

Reply:
both pages should also have some padding on the left and right side of the screen as depicted on the design